components/sync/sources.gni

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+brave_components_sync_deps = [ "//brave/components/brave_sync:features" ]
```

Comment:
should be `brave_components_sync_public_deps`

Reply:
fixed at https://github.com/brave/brave-core/pull/16478/commits/de714a0457310fb6a10f47c1f626519a1e7287c9